content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Minikube to Create a Cluster
+weight: 10
+---
+
+## {{% heading "objectives" %}}
+
+* Learn what a Kubernetes cluster is.
+* Learn what Minikube is.
+* Start a Kubernetes cluster on your computer.
+
+## Kubernetes Clusters
+
+<div class="col-md-8">
+  <div class="content__box content__box_fill">
+    <p><i>Kubernetes is a production-grade, open-source platform that orchestrates
+      the placement (scheduling) and execution of application containers
+      within and across computer clusters.</i></p>
+  </div>
+</div>
+ 
+**Kubernetes coordinates a highly available cluster of computers that are connected
+to work as a single unit.** The abstractions in Kubernetes allow you to deploy
+containerized applications to a cluster without tying them specifically to individual
+machines. To make use of this new model of deployment, applications need to be packaged
+in a way that decouples them from individual hosts: they need to be containerized.
+Containerized applications are more flexible and available than in past deployment models,
+where applications were installed directly onto specific machines as packages deeply
+integrated into the host. **Kubernetes automates the distribution and scheduling of
+application containers across a cluster in a more efficient way.** Kubernetes is an
+open-source platform and is production-ready.
+
+A Kubernetes cluster consists of two types of resources:
+
+* The **Control Plane** coordinates the cluster
+* **Nodes** are the workers that run applications
+
+### Cluster Diagram
+
+{{< figure src="../../public/images/module_01_cluster.svg" class="diagram-medium" >}}
+
+**The Control Plane is responsible for managing the cluster.** The Control Plane
+coordinates all activities in your cluster, such as scheduling applications, maintaining
+applications' desired state, scaling applications, and rolling out new updates.
+
+<div class="col-md-8">
+  <div style="background-color: #eaf7ff; border-left: 5px solid #3771e3; padding: 15px; margin: 20px 0;">
+    <p><i>Control Planes manage the cluster and the nodes that are used to host the running applications.</i></p>
+  </div>
+</div>
```

Comment:
Is it feasible to use a Markdown paragraph with a custom class (and some associated CSS)?

Reply:
I've used the `{{% alert %}}` suggestion.